netbox/utilities/testing/views.py

```diff
@@ -583,6 +598,55 @@ def test_bulk_import_objects_with_permission(self):
             self.assertHttpStatus(self.client.post(self._get_url('import'), data), 200)
             self.assertEqual(self._get_queryset().count(), initial_count + len(self.csv_data) - 1)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=['*'])
+        def test_bulk_update_objects_with_permission(self):
+            if not self.csv_update_data:
```

Comment:
We probably want to raise an error if `csv_update_data` hasn't been set, as it likely means that someone forgot to add it.